tests/test_visualizer.py

```diff
@@ -0,0 +1,49 @@
+"""
+Tests for the visualizer module
+"""
+
+import pytest
+from matplotlib import pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+from pathlib import Path
+
+from ppinsight import visualizer
+
+# setup variables
+models = ['../dummy_scores/model_1.csv','/home/fmclary/CSE583/PPInsight/dummy_scores/model_2.csv','/home/fmclary/CSE583/PPInsight/dummy_scores/model_3.csv']
+
+def test_not_a_list():
+    """Edge test to ensure function throws TypeError if passed a non-list object"""
+    with pytest.raises(
+        TypeError, match='Format your input as a list of file paths'
+    ):
+        to_plot('../dummy_scores/model_1.csv','../dummy_scores/model_2.csv')
```

Comment:
@mayagatt @fmclary 

Seems like `'../dummy_scores/model_1.csv','../dummy_scores/model_2.csv'` could (in the future) refer to `models[0]` and `models[1]`. That way if you move things around your code won't break in random places. I'm just calling it out so you know what to look for in the future